README.md

```diff
@@ -1 +1,6 @@
-# GAUSS_EDA_PROJECT
\ No newline at end of file
+# GAUSS_EDA_PROJECT
+
+Requirements:
+- A valid Kaggle API key stored in the file 'kaggle.json' in the directory: `~/.kaggle/` (for Unix-like systems) or `%USERPROFILE%\.kaggle\` (for Windows).
+- The dataset name should be specified in the .env file under the key 'DATASET'.
+- Files to be deleted should be passed as a list in the 'delete_files' argument or specified in the .env file under the key 'DELETE_FILES', separated by commas.
```

Comment:
If the user will read this, he will not understand the meaning. What should be passed? To what? This file just describes the project.

You can make a section with the help of markdown. For example:

## Installation
* A valid Kaggle API credentials stored in the file 'kaggle.json' in the directory: `~/.kaggle/` (for Unix-like systems) or `%USERPROFILE%\.kaggle\` (for Windows). You can read about that in ....

* `.env` file contents
```
DATASET="Add the name here"
```

Reply:
Making a proper readme file is a part of #1 .
I will slightly improve the installation text, but readme file will still have to be fully done.